### PR TITLE
Se 466 access level withdraw rule documentation

### PR DESCRIPTION
**Pull request by TJ-Everett**

### CODE REVIEW

File: docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+# Withdrawal Limit By Access Level Rule
+
+## Purpose
+
+The purpose of this rule is to provide withdrawal limits for accounts at the application level based on an application defined segment of users. The segments are defined as the access levels of the accounts. This rule may be used to provide gated withdrawal limits of assets to ensure accounts cannot withdraw more US Dollars or chain native tokens without first performing other actions defined by the application. For example, the application may decide users may not withdraw without performing specific onboarding activities. The application developer may set a maximum withdraw limit of $0 for the default access level and $1000 for the next access level. As accounts are introduced to the ecosystem, they may not withdraw from the ecosystem until the application changes their access level to a higher value. This rule does not prevent the accumulation of protocol supported assets.
```
Comment (oscarsernarosero) [inline docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md]: withdrawal here seems too vague of a term. Maybe swapping the first `withdrawal limits for accounts at the application level ` for `limits on the amount of funds that an account can take out from the application economy` or something like that.

`a maximum withdraw limit of $0` could be a bit confusing. Maybe adding *restrictive*... `the most restrictive withdrawal limit of $0`

File: docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+The rule will be evaluated with the following logic:
+
+1. The application manager sends to the protocol's rule processor the dollar value sum of all application assets the account has already withdrawn, the access level of the account, the ruleId, and the dollar amount to be transferred in the transaction.
+2. The processor retrieves the maximum withdrawal limit allowed for the rule with the ruleId passed, and for the access level of the account. If the withdrawal amount plus already withdrawn amount exceeds the maximum allowed by the rule in the case of a successful transactions, then the transaction reverts.
+
+###### *see [ApplicationAccessLevelProcessorFacet](../../../src/protocol/economic/ruleProcessor/ApplicationAccessLevelProcessorFacet.sol) -> checkeithdrawalLimitsByAccessLevel*
+
+## Evaluation Exceptions 
+- This rule doesn't apply when an **ruleBypassAccount** address is in either the *from* or the *to* side of the transaction. This doesn't necessarily mean that if an app administrator is the one executing the transaction it will bypass the rule, unless the aforementioned condition is true.
```
Comment (oscarsernarosero) [inline docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md]: we need to fix the `... if an app administrator is ...` for `... if a rule bypasser account is ...` in all the files. I overlooked that one.

Comment (oscarsernarosero) [reply docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md]: Also `... apply when an **ruleBypassAccount** ` for `... apply when a **ruleBypassAccount** `

**Review by oscarsernarosero — APPROVED** [review]
Looks really good Tayler. I have very small suggestions.

File: docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+### Parameter Optionality:
+
+There are no options for the parameters of this rule.
+
+### Parameter Validation:
+
+The following validation will be carried out by the create function in order to ensure that these parameters are valid and make sense:
+
+- The `_withdrawalAmounts` array has length 5.
+- The elements of the `_withdrawalAmounts` array are in ascendant order.
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/WITHDRAWAL-LIMIT-BY-ACCESS-LEVEL.md]: "ascendant" seems a little weird. Can we just say, "ascending"?

**Review by ShaneDuncan602 — APPROVED** [review]
You're getting very good at these. Approved with one small comment.

**Pull request merged**